src/wp-includes/load.php

```diff
@@ -1806,8 +1806,13 @@ function wp_start_scraping_edited_file_errors() {
 
 	$key   = substr( sanitize_key( wp_unslash( $_REQUEST['wp_scrape_key'] ) ), 0, 32 );
 	$nonce = wp_unslash( $_REQUEST['wp_scrape_nonce'] );
+	$transient = get_transient( 'scrape_key_' . $key );
 
-	if ( get_transient( 'scrape_key_' . $key ) !== $nonce ) {
+	if ( $transient === false ) {
```

Comment:
```suggestion
	if ( false === $transient ) {
```

Use Yoda condition check.